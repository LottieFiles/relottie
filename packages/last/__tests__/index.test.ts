/**
 * Copyright 2023 Design Barn Inc.
 */

import { TITLES } from '../src/titles.js';

const SNAPSHOT_DIR_PATH = './__snapshots__/';

test('TITLES should match snapshots', () => {
  expect(TITLES).toMatchSnapshot(`${SNAPSHOT_DIR_PATH}-titles.snap`);
});
