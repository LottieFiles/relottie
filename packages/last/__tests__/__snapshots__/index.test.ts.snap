// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TITLES should match snapshots: ./__snapshots__/-titles.snap 1`] = `
Object {
  "array": Object {
    "outTangentsValue": "out-tangents-value",
  },
  "boolean": Object {
    "bezierClosed": "bezier-closed",
    "collapseTransform": "collapse-transform",
    "hasMask": "has-mask",
    "hidden": "hidden",
    "inverted": "inverted",
    "motionBlurEnabled": "motion-blur-enabled",
    "splitEnabled": "split-enabled",
    "strokeOverFill": "stroke-over-fill",
  },
  "collection": Object {
    "assets": "assets",
    "bezierInTangents": "bezier-in-tangents",
    "bezierOutTangents": "bezier-out-tangents",
    "bezierVertices": "bezier-vertices",
    "bezierXAxis": "bezier-x-axis",
    "bezierYAxis": "bezier-y-axis",
    "colorRgba": "color-rgba",
    "composition": "composition",
    "documentFillColor": "document-fill-color",
    "documentStrokeColor": "document-stroke-color",
    "effectList": "effect-list",
    "effectParamList": "effect-param-list",
    "extraCompositions": "extra-compositions",
    "keyframeEndValue": "keyframe-end-value",
    "keyframeList": "keyframe-list",
    "keyframeValue": "keyframe-value",
    "keywords": "keywords",
    "layerStyle": "layer-style",
    "layerStyleList": "layer-style-list",
    "markers": "markers",
    "masksProperties": "masks-properties",
    "positionKeyframeInTangents": "position-keyframe-in-tangents",
    "positionKeyframeList": "position-keyframe-list",
    "positionKeyframeOutTangents": "position-keyframe-out-tangents",
    "shapeKeyframeList": "shape-keyframe-list",
    "shapeKeyframeStart": "shape-keyframe-start",
    "shapeList": "shape-list",
    "staticValues": "static-values",
    "strokeDashList": "stroke-dash-list",
    "textCharacters": "text-characters",
    "textDocumentKeyframes": "text-document-keyframes",
    "textFontList": "text-font-list",
    "textRanges": "text-ranges",
    "wrapPosition": "wrap-position",
    "wrapSize": "wrap-size",
  },
  "custom": "custom",
  "element": Object {
    "align": "align",
    "altitude": "altitude",
    "amount": "amount",
    "anchorPoint": "anchor-point",
    "angle": "angle",
    "animatedColorProp": "animated-color-prop",
    "animatedGradientColors": "animated-gradient-colors",
    "animatedMultidimensionalProp": "animated-multidimensional-prop",
    "animatedPositionProp": "animated-position-prop",
    "animatedShapeBezier": "animated-shape-bezier",
    "animatedShapeProp": "animated-shape-prop",
    "animatedValueProp": "animated-value-prop",
    "audioLevel": "audio-level",
    "bevelStyle": "bevel-style",
    "blurSize": "blur-size",
    "chokeSpread": "choke-spread",
    "copies": "copies",
    "dilate": "dilate",
    "distance": "distance",
    "effectValueAngle": "effect-value-angle",
    "effectValueCheckbox": "effect-value-checkbox",
    "effectValueColor": "effect-value-color",
    "effectValueDropdown": "effect-value-dropdown",
    "effectValueLayer": "effect-value-layer",
    "effectValuePoint": "effect-value-point",
    "effectValueSlider": "effect-value-slider",
    "firstMargin": "first-margin",
    "forceAlignment": "force-alignment",
    "globalAngle": "global-angle",
    "gradientOverlayScale": "gradient-overlay-scale",
    "highlightAngle": "highlight-angle",
    "highlightColor": "highlight-color",
    "highlightLength": "highlight-length",
    "highlightMode": "highlight-mode",
    "highlightOpacity": "highlight-opacity",
    "innerRadius": "inner-radius",
    "innerRoundness": "inner-roundness",
    "invert": "invert",
    "jitter": "jitter",
    "keyframeBezierHandleInTangent": "keyframe-bezier-handle-in-tangent",
    "keyframeBezierHandleOutTangent": "keyframe-bezier-handle-out-tangent",
    "keyframeInTangent": "keyframe-in-tangent",
    "keyframeListInTangent": "keyframe-list-in-tangent",
    "keyframeListOutTangent": "keyframe-list-out-tangent",
    "keyframeOutTangent": "keyframe-out-tangent",
    "keyframeValueInTangent": "keyframe-value-in-tangent",
    "keyframeValueOutTangent": "keyframe-value-out-tangent",
    "lastMargin": "last-margin",
    "layerAudioSettings": "layer-audio-settings",
    "layerConceal": "layer-conceal",
    "layerStyleBlendMode": "layer-style-blend-mode",
    "layerStyleColor": "layer-style-color",
    "layerStyleOpacity": "layer-style-opacity",
    "layerStyleStrokeSize": "layer-style-stroke-size",
    "layerTransform": "layer-transform",
    "layerTransformOrientation": "layer-transform-orientation",
    "layerTransformScale": "layer-transform-scale",
    "layerTransformSkew": "layer-transform-skew",
    "layerTransformSkewAxis": "layer-transform-skew-axis",
    "letterSpacing": "letter-spacing",
    "lineSpacing": "line-spacing",
    "maskOpacity": "mask-opacity",
    "maskVertices": "mask-vertices",
    "metadata": "metadata",
    "miterLimit": "miter-limit",
    "miterLimitAlternative": "miter-limit-alternative",
    "motionBlur": "motion-blur",
    "noise": "noise",
    "perpendicularToPath": "perpendicular-to-path",
    "perspective": "perspective",
    "polygonRotationClockwise": "polygon-rotation-clockwise",
    "positionKeyframeInTangent": "position-keyframe-in-tangent",
    "positionKeyframeListInTangent": "position-keyframe-list-in-tangent",
    "positionKeyframeListOutTangent": "position-keyframe-list-out-tangent",
    "positionKeyframeOutTangent": "position-keyframe-out-tangent",
    "radius": "radius",
    "range": "range",
    "reverse": "reverse",
    "reversePath": "reverse-path",
    "rotationClockwise": "rotation-clockwise",
    "rotationX": "rotation-x",
    "rotationY": "rotation-y",
    "rotationZ": "rotation-z",
    "rounded": "rounded",
    "roundness": "roundness",
    "selectorSmoothness": "selector-smoothness",
    "shadowColor": "shadow-color",
    "shadowMode": "shadow-mode",
    "shadowOpacity": "shadow-opacity",
    "shape": "shape",
    "shapeEllipseSize": "shape-ellipse-size",
    "shapeFillColor": "shape-fill-color",
    "shapeFillOpacity": "shape-fill-opacity",
    "shapeGradientFillEndPoint": "shape-gradient-fill-end-point",
    "shapeGradientFillOpacity": "shape-gradient-fill-opacity",
    "shapeGradientFillRule": "shape-gradient-fill-rule",
    "shapeGradientFillStartPoint": "shape-gradient-fill-start-point",
    "shapeGradientStrokeEndPoint": "shape-gradient-stroke-end-point",
    "shapeGradientStrokeOpacity": "shape-gradient-stroke-opacity",
    "shapeGradientStrokeStartPoint": "shape-gradient-stroke-start-point",
    "shapeKeyframeInTangent": "shape-keyframe-in-tangent",
    "shapeKeyframeOutTangent": "shape-keyframe-out-tangent",
    "shapePolygonOuterRadius": "shape-polygon-outer-radius",
    "shapePolygonOuterRoundness": "shape-polygon-outer-roundness",
    "shapePolygonPoints": "shape-polygon-points",
    "shapeRectangleSize": "shape-rectangle-size",
    "shapeStarOuterRadius": "shape-star-outer-radius",
    "shapeStarOuterRoundness": "shape-star-outer-roundness",
    "shapeStarPoints": "shape-star-points",
    "shapeStrokeColor": "shape-stroke-color",
    "shapeStrokeOpacity": "shape-stroke-opacity",
    "shapeTransformOrientation": "shape-transform-orientation",
    "shapeTransformScale": "shape-transform-scale",
    "shapeTransformSkew": "shape-transform-skew",
    "shapeTransformSkewAxis": "shape-transform-skew-axis",
    "shapeTrimCenter": "shape-trim-center",
    "shapeTrimEnd": "shape-trim-end",
    "shapeTrimOffset": "shape-trim-offset",
    "shapeTrimStart": "shape-trim-start",
    "shapeZigZagSize": "shape-zig-zag-size",
    "shapeZigZagsPoints": "shape-zig-zags-points",
    "smoothness": "smoothness",
    "soften": "soften",
    "source": "source",
    "splitX": "split-x",
    "splitY": "split-y",
    "splitZ": "split-z",
    "startOpacity": "start-opacity",
    "strength": "strength",
    "strokeBrightness": "stroke-brightness",
    "strokeColor": "stroke-color",
    "strokeHue": "stroke-hue",
    "strokeLength": "stroke-length",
    "strokeOpacity": "stroke-opacity",
    "strokeSaturation": "stroke-saturation",
    "strokeWidth": "stroke-width",
    "technique": "technique",
    "textAlignmentOptions": "text-alignment-options",
    "textAnimatedDocument": "text-animated-document",
    "textAnimatorData": "text-animator-data",
    "textBlur": "text-blur",
    "textCharacterData": "text-character-data",
    "textData": "text-data",
    "textDocument": "text-document",
    "textFillBrightness": "text-fill-brightness",
    "textFillColor": "text-fill-color",
    "textFillHue": "text-fill-hue",
    "textFillOpacity": "text-fill-opacity",
    "textFillSaturation": "text-fill-saturation",
    "textFollowPath": "text-follow-path",
    "textFonts": "text-fonts",
    "textGroupAlignment": "text-group-alignment",
    "textMaxAmount": "text-max-amount",
    "textMaxEase": "text-max-ease",
    "textMinEase": "text-min-ease",
    "textSelector": "text-selector",
    "textSelectorEnd": "text-selector-end",
    "textSelectorStart": "text-selector-start",
    "textStyle": "text-style",
    "textStyleOrientation": "text-style-orientation",
    "textStyleScale": "text-style-scale",
    "textStyleSkew": "text-style-skew",
    "textStyleSkewAxis": "text-style-skew-axis",
    "timeRemapping": "time-remapping",
    "transformOpacity": "transform-opacity",
    "transformRepeater": "transform-repeater",
    "transformRepeaterOrientation": "transform-repeater-orientation",
    "transformRepeaterScale": "transform-repeater-scale",
    "transformRepeaterSkew": "transform-repeater-skew",
    "transformRepeaterSkewAxis": "transform-repeater-skew-axis",
    "translation": "translation",
    "userMetadata": "user-metadata",
  },
  "intBoolean": Object {
    "animated": "animated",
    "autoOrient": "auto-orient",
    "collapseTransformNew": "collapse-transform-new",
    "embedded": "embedded",
    "enabled": "enabled",
    "expressible": "expressible",
    "extraComposition": "extra-composition",
    "hold": "hold",
    "keyframeBezierHandleXAxis": "keyframe-bezier-handle-x-axis",
    "keyframeBezierHandleYAxis": "keyframe-bezier-handle-y-axis",
    "keyframeListXAxis": "keyframe-list-x-axis",
    "keyframeListYAxis": "keyframe-list-y-axis",
    "keyframeValueXAxis": "keyframe-value-x-axis",
    "keyframeValueYAxis": "keyframe-value-y-axis",
    "keyframeXAxis": "keyframe-x-axis",
    "keyframeYAxis": "keyframe-y-axis",
    "layerThreedimensional": "layer-threedimensional",
    "positionKeyframeListXAxis": "position-keyframe-list-x-axis",
    "positionKeyframeListYAxis": "position-keyframe-list-y-axis",
    "positionKeyframeXAxis": "position-keyframe-x-axis",
    "positionKeyframeYAxis": "position-keyframe-y-axis",
    "randomize": "randomize",
    "threedimensional": "threedimensional",
  },
  "number": Object {
    "adaptiveSampleLimit": "adaptive-sample-limit",
    "ascent": "ascent",
    "baselineShift": "baseline-shift",
    "blendMode": "blend-mode",
    "blendModeColor": "blend-mode-color",
    "blendModeColorBurn": "blend-mode-color-burn",
    "blendModeColorDodge": "blend-mode-color-dodge",
    "blendModeDarken": "blend-mode-darken",
    "blendModeDifference": "blend-mode-difference",
    "blendModeExclusion": "blend-mode-exclusion",
    "blendModeHardLight": "blend-mode-hard-light",
    "blendModeHue": "blend-mode-hue",
    "blendModeLighten": "blend-mode-lighten",
    "blendModeLuminosity": "blend-mode-luminosity",
    "blendModeMultiply": "blend-mode-multiply",
    "blendModeNormal": "blend-mode-normal",
    "blendModeOverlay": "blend-mode-overlay",
    "blendModeSaturation": "blend-mode-saturation",
    "blendModeScreen": "blend-mode-screen",
    "blendModeSoftLight": "blend-mode-soft-light",
    "composite": "composite",
    "compositeAbove": "composite-above",
    "compositeBelow": "composite-below",
    "compositionIndex": "composition-index",
    "count": "count",
    "dataSourceType": "data-source-type",
    "effectPropertyCount": "effect-property-count",
    "effectType": "effect-type",
    "effectValueIgnored": "effect-value-ignored",
    "effectValueType": "effect-value-type",
    "expressionPropertyIndex": "expression-property-index",
    "fillRule": "fill-rule",
    "fillRuleEvenOdd": "fill-rule-even-odd",
    "fillRuleNonZero": "fill-rule-non-zero",
    "fontPathOrigin": "font-path-origin",
    "fontPathOriginCssUrl": "font-path-origin-css-url",
    "fontPathOriginFonturl": "font-path-origin-fonturl",
    "fontPathOriginLocal": "font-path-origin-local",
    "fontPathOriginScriptUrl": "font-path-origin-script-url",
    "fontSize": "font-size",
    "framerate": "framerate",
    "gradientType": "gradient-type",
    "gradientTypeLinear": "gradient-type-linear",
    "gradientTypeRadial": "gradient-type-radial",
    "height": "height",
    "inPoint": "in-point",
    "keyframeBezierHandleTime": "keyframe-bezier-handle-time",
    "keyframeListTime": "keyframe-list-time",
    "keyframeTime": "keyframe-time",
    "keyframeValueTime": "keyframe-value-time",
    "layerStyleType": "layer-style-type",
    "layerType": "layer-type",
    "length": "length",
    "lineCap": "line-cap",
    "lineCapButt": "line-cap-butt",
    "lineCapRound": "line-cap-round",
    "lineCapSquare": "line-cap-square",
    "lineHeight": "line-height",
    "lineJoin": "line-join",
    "lineJoinBevel": "line-join-bevel",
    "lineJoinMiter": "line-join-miter",
    "lineJoinRound": "line-join-round",
    "markerDuration": "marker-duration",
    "markerTime": "marker-time",
    "matteMode": "matte-mode",
    "matteModeAlpha": "matte-mode-alpha",
    "matteModeInvertedAlpha": "matte-mode-inverted-alpha",
    "matteModeInvertedLuma": "matte-mode-inverted-luma",
    "matteModeLuma": "matte-mode-luma",
    "matteModeNormal": "matte-mode-normal",
    "matteParent": "matte-parent",
    "matteTarget": "matte-target",
    "mergeMode": "merge-mode",
    "miterLimitValue": "miter-limit-value",
    "numberOfProperties": "number-of-properties",
    "outPoint": "out-point",
    "parentIndex": "parent-index",
    "positionKeyframeListTime": "position-keyframe-list-time",
    "positionKeyframeTime": "position-keyframe-time",
    "propertyIndex": "property-index",
    "rangeUnits": "range-units",
    "rangeUnitsIndex": "range-units-index",
    "rangeUnitsPercent": "range-units-percent",
    "samplesPerFrame": "samples-per-frame",
    "shapeDirection": "shape-direction",
    "shapeDirectionClockwise": "shape-direction-clockwise",
    "shapeDirectionClockwiseReversed": "shape-direction-clockwise-reversed",
    "shapeDirectionNormal": "shape-direction-normal",
    "shapeKeyframeTime": "shape-keyframe-time",
    "shapePathIndex": "shape-path-index",
    "shapePolygonStarType": "shape-polygon-star-type",
    "shutterAngle": "shutter-angle",
    "shutterPhase": "shutter-phase",
    "staticValue": "static-value",
    "textBased": "text-based",
    "textBasedCharacterExcludingSpaces": "text-based-character-excluding-spaces",
    "textBasedCharacters": "text-based-characters",
    "textBasedLines": "text-based-lines",
    "textBasedWords": "text-based-words",
    "textCaps": "text-caps",
    "textCapsAll": "text-caps-all",
    "textCapsRegular": "text-caps-regular",
    "textCapsSmall": "text-caps-small",
    "textGrouping": "text-grouping",
    "textGroupingAll": "text-grouping-all",
    "textGroupingCharacters": "text-grouping-characters",
    "textGroupingLine": "text-grouping-line",
    "textGroupingWords": "text-grouping-words",
    "textJustiftyCenter": "text-justifty-center",
    "textJustiftyLeft": "text-justifty-left",
    "textJustiftyRight": "text-justifty-right",
    "textJustiftyWithLastLineCenter": "text-justifty-with-last-line-center",
    "textJustiftyWithLastLineFull": "text-justifty-with-last-line-full",
    "textJustiftyWithLastLineLeft": "text-justifty-with-last-line-left",
    "textJustiftyWithLastLineRight": "text-justifty-with-last-line-right",
    "textJustify": "text-justify",
    "textMask": "text-mask",
    "textRangeSelectorMode": "text-range-selector-mode",
    "textShape": "text-shape",
    "textShapeRampDown": "text-shape-ramp-down",
    "textShapeRampUp": "text-shape-ramp-up",
    "textShapeRound": "text-shape-round",
    "textShapeSmooth": "text-shape-smooth",
    "textShapeSquare": "text-shape-square",
    "textShapeTriangle": "text-shape-triangle",
    "textTracking": "text-tracking",
    "textVerticalJustify": "text-vertical-justify",
    "textVerticalJustifyBottom": "text-vertical-justify-bottom",
    "textVerticalJustifyCenter": "text-vertical-justify-center",
    "textVerticalJustifyTop": "text-vertical-justify-top",
    "timeStart": "start-time",
    "timeStretch": "time-stretch",
    "trimMultipleShapes": "trim-multiple-shapes",
    "trimMultipleShapesIndividually": "trim-multiple-shapes-individually",
    "trimMultipleShapesSimultaneously": "trim-multiple-shapes-simultaneously",
    "width": "width",
  },
  "object": Object {
    "animatedColor": "animated-color",
    "animatedColorStatic": "animated-color-static",
    "animatedMultidimensional": "animated-multidimensional",
    "animatedMultidimensionalStatic": "animated-multidimensional-static",
    "animatedPosition": "animated-position",
    "animatedPositionStatic": "animated-position-static",
    "animatedShape": "animated-shape",
    "animatedShapeStatic": "animated-shape-static",
    "animatedValue": "animated-value",
    "animatedValueStatic": "animated-value-static",
    "animation": "animation",
    "assetDataSource": "asset-data-source",
    "assetFile": "asset-file",
    "assetImage": "asset-image",
    "assetPrecomposition": "asset-precomposition",
    "bezier": "bezier",
    "characterData": "character-data",
    "characterPrecomp": "character-precomp",
    "characterShapes": "character-shapes",
    "effectCustom": "effect-custom",
    "effectDisplacementMap": "effect-displacement-map",
    "effectDropShadow": "effect-drop-shadow",
    "effectFill": "effect-fill",
    "effectGaussianBlur": "effect-gaussian-blur",
    "effectMatte3": "effect-matte3",
    "effectMeshWarp": "effect-mesh-warp",
    "effectPaintOverTransparent": "effect-paint-over-transparent",
    "effectParamAngle": "effect-param-angle",
    "effectParamCheckbox": "effect-param-checkbox",
    "effectParamColor": "effect-param-color",
    "effectParamDropdown": "effect-param-dropdown",
    "effectParamIgnored": "effect-param-ignored",
    "effectParamLayer": "effect-param-layer",
    "effectParamNoValue": "effect-param-no-value",
    "effectParamPoint": "effect-param-point",
    "effectParamSlider": "effect-param-slider",
    "effectProLevels": "effect-pro-levels",
    "effectPuppet": "effect-puppet",
    "effectRadialWipe": "effect-radial-wipe",
    "effectSpherize": "effect-spherize",
    "effectStroke": "effect-stroke",
    "effectTint": "effect-tint",
    "effectTritone": "effect-tritone",
    "effectWavy": "effect-wavy",
    "endOpacity": "end-opacity",
    "gradientOverlay": "gradient-overlay",
    "keyframe": "keyframe",
    "keyframeBezierHandle": "keyframe-bezier-handle",
    "layerAdjustment": "layer-adjustment",
    "layerAudio": "layer-audio",
    "layerCamera": "layer-camera",
    "layerData": "layer-data",
    "layerGuide": "layer-guide",
    "layerImage": "layer-image",
    "layerImagePlaceholder": "layer-image-placeholder",
    "layerImageSequence": "layer-image-sequence",
    "layerLight": "layer-light",
    "layerNull": "layer-null",
    "layerPrecomposition": "layer-precomposition",
    "layerShape": "layer-shape",
    "layerSolidColor": "layer-solid-color",
    "layerStyleBevelEmboss": "layer-style-bevel-emboss",
    "layerStyleChokeSpread": "layer-style-choke-spread",
    "layerStyleColorOverlay": "layer-style-color-overlay",
    "layerStyleDropShadow": "layer-style-drop-shadow",
    "layerStyleGradientOverlay": "layer-style-gradient-overlay",
    "layerStyleInnerGlow": "layer-style-inner-glow",
    "layerStyleInnerShadow": "layer-style-inner-shadow",
    "layerStyleOuterGlow": "layer-style-outer-glow",
    "layerStyleSatin": "layer-style-satin",
    "layerStyleStroke": "layer-style-stroke",
    "layerText": "layer-text",
    "layerVideo": "layer-video",
    "layerVideoPlaceholder": "layer-video-placeholder",
    "marker": "marker",
    "mask": "mask",
    "positionKeyframe": "position-keyframe",
    "shapeEllipse": "shape-ellipse",
    "shapeFill": "shape-fill",
    "shapeGradientFill": "shape-gradient-fill",
    "shapeGradientStroke": "shape-gradient-stroke",
    "shapeGroup": "shape-group",
    "shapeKeyframe": "shape-keyframe",
    "shapeMerge": "shape-merge",
    "shapeOffsetPath": "shape-offset-path",
    "shapePath": "shape-path",
    "shapePolyStar": "shape-poly-star",
    "shapePolygon": "shape-polygon",
    "shapePuckerBloat": "shape-pucker-bloat",
    "shapeRectangle": "shape-rectangle",
    "shapeRepeater": "shape-repeater",
    "shapeRoundedCorners": "shape-rounded-corners",
    "shapeStar": "shape-star",
    "shapeStroke": "shape-stroke",
    "shapeTransform": "shape-transform",
    "shapeTrim": "shape-trim",
    "shapeTwist": "shape-twist",
    "shapeZigZags": "shape-zig-zag",
    "splitVector": "split-vector",
    "strokeDashDefault": "stroke-dash-default",
    "strokeDashGap": "stroke-dash-gap",
    "strokeDashLength": "stroke-dash-length",
    "strokeDashOffset": "stroke-dash-offset",
    "textDocumentKeyframe": "text-document-keyframe",
    "textFont": "text-font",
    "textRange": "text-range",
  },
  "string": Object {
    "assetImageType": "asset-image-type",
    "assetType": "asset-type",
    "author": "author",
    "character": "character",
    "cssClass": "css-class",
    "description": "description",
    "expression": "expression",
    "filename": "filename",
    "fontCssClass": "font-css-class",
    "fontFamily": "font-family",
    "fontName": "font-name",
    "fontPath": "font-path",
    "fontStyle": "font-style",
    "fontWeight": "font-weight",
    "generator": "generator",
    "hexColor": "hex-color",
    "idAssetDataSource": "id-asset-data-source",
    "idAssetFile": "id-asset-file",
    "idAssetImage": "id-asset-image",
    "idAssetPrecomposition": "id-asset-precomposition",
    "idDataSource": "data-source-id",
    "idImage": "image-id",
    "idLayerXml": "layer-xml-id",
    "idReference": "reference-id",
    "idSound": "sound-id",
    "keyword": "keyword",
    "layerXmlTagName": "layer-xml-tag-name",
    "markerComment": "marker-comment",
    "maskMode": "mask-mode",
    "maskModeAdd": "mask-mode-add",
    "maskModeDarken": "mask-mode-darken",
    "maskModeDifference": "mask-mode-difference",
    "maskModeIntersect": "mask-mode-intersect",
    "maskModeLighten": "mask-mode-lighten",
    "maskModeNo": "mask-mode-no",
    "maskModeSubtract": "mask-mode-subtract",
    "matchName": "match-name",
    "name": "name",
    "path": "path",
    "shapeType": "shape-type",
    "strokeDashType": "stroke-dash-type",
    "text": "text",
    "themeColor": "theme-color",
    "version": "version",
  },
}
`;
